docs/.vuepress/config.js

```diff
@@ -1,78 +1,101 @@
+const glob = require("glob");
+const markdownIt = require("markdown-it");
+const meta = require("markdown-it-meta");
+const fs = require("fs");
+const _ = require("lodash");
+
+const sidebar = (directory, array) => {
+  return array.map(i => {
+    const children = _.sortBy(
+      glob
+        .sync(`./${directory}/${i[1]}/*.md`)
+        .map(path => {
+          const md = new markdownIt();
+          const file = fs.readFileSync(path, "utf8");
+          md.use(meta);
+          md.render(file);
+          const order = md.meta.order;
+          return { path, order };
+        })
+        .filter(f => f.order !== false),
+      ["order", "path"]
+    )
+      .map(f => f.path)
+      .filter(f => !f.match("readme"));
+    return {
+      title: i[0],
+      children
+    };
+  });
+};
+
 module.exports = {
-  title: "Cosmos SDK Documentation",
-  description: "Documentation for the Cosmos SDK and Gaia.",
-  ga: "UA-51029217-2",
-  dest: "./dist/docs",
-  base: "/docs/",
-  markdown: {
-    lineNumbers: true
+  title: "Cosmos SDK",
+  base: process.env.VUEPRESS_BASE || "/",
+  locales: {
+    "/": {
+      lang: "en-US"
+    },
+    "/ru/": {
+      lang: "ru"
+    },
+    // "/kr/": {
```

Comment:
this shouldn't be commented. Please delete or uncomment

Reply:
Deleted.